airbyte-integrations/connectors/source-hubspot/unit_tests/integrations/test_engagements_calls.py

```diff
@@ -68,7 +68,7 @@ def test_given_oauth_authentication_when_read_then_perform_authenticated_queries
             http_mocker,
             with_oauth=True,
             with_dynamic_schemas=True,
-            entities=OBJECTS_WITH_DYNAMIC_SCHEMA,
+            entities=["calls"],
```

Comment:
Before in v6 CDK we would evaluate all the json schemas when we call `ConcurrentDeclarativeSource.schemas()` which required that we mock every dynamic schema stream. But now in CDK v7 we lazily retrieve the json schema and make the outbound API requests for the streams defined in the catalog.

So now we don't need to mock every `/properties` call and if we did, this test would fail with mocks that were not triggered